pycrdt_websocket/yroom.py

```diff
@@ -230,6 +234,11 @@ async def stop(self) -> None:
         self._stopped.set()
         self._task_group.cancel_scope.cancel()
         self._task_group = None
+        if self.ystore is not None:
+            try:
+                await self.ystore.stop()
+            except RuntimeError:
+                pass
```

Comment:
I'm not sure we should automatically stop the YStore when the YRoom is stopped.
`WebsocketServer` has an `auto_clean_room` parameter, maybe there should be an `auto_stop_store` as well, that we would use to do so if set to `True`.
Also, we should have YStore use the same exception handler pattern that we used for `YRoom` and `WebsocketServer`, and not catch exceptions here.

Reply:
Do we want to make it configurable? Sure. And also +1 for adding exception handler pattern in other logic to protect task group. In this case, we are trying to except a RuntimeError that is thrown if ystore is already stopped.